packages/lit-dev-content/site/docs/composition/controllers.md

```diff
@@ -70,8 +72,23 @@ class ClockController implements ReactiveController {
 }
 ```
 
+```js
+class ClockController {
+  constructor(host) {
+    // Store a reference to the host
+    this.host = host;
```

Comment:
I wonder if we want to use class fields in JS examples? They're supported in all current browsers now.

Reply:
In this case do you mean?:

```js
class ClockController {
  host;
  constructor(host) {
    // Store a reference to the host
    this.host = host;
    // Register for lifecycle updates
    host.addController(this);
  }
}
```

Reply:
Yeah. Though I'm not even sure myself whether that's better or not...

Reply:
I think I'll leave it off for now, but can revisit.

Reply:
Or may be more simple: 
```js
class ClockController {
  constructor(private host) {
    // Register for lifecycle updates
    host.addController(this);
  }
}
```